app/src/main/java/com/example/firstapplication/UserListViewModel.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.firstapplication
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class UserListViewModel: ViewModel() {
+
+    val users: MutableLiveData<List<User>> by lazy {
+        MutableLiveData<List<User>>()
+    }
+
+    init {
+        users.postValue(mutableListOf(
+            User("user-id-1", "user-name-1", "user-1-email"),
+            User("user-id-2", "user-name-2", "user-2-email"),
+            User("user-id-3", "user-name-3", "user-3-email"),
+            User("user-id-4", "user-name-4", "user-4-email"),
+            User("user-id-5", "user-name-5", "user-5-email")
+        ))
+    }
+
+    fun add(user: User) {
+        val currentList = users.value
+        if (currentList == null) {
```

Comment:
Fajnie że o tym pomyślałaś, choć w twoim kodzie nie powinno to wystąpić, bo wstawiasz początkową wartość w inicie

Reply:
to prawda :) jednak zostawię to na razie bo z tego co widziałam w dalszych krokach jest korzystanie z zewnętrznego API